apps/settings/elements/developer_hud.html

```diff
@@ -9,6 +9,10 @@ <h1 data-l10n-id="hud-settings-header"></h1>
       <ul>
         <li>
           <label class="pack-checkbox">
```

Comment:
Drive-by-nit: If you want to land this before the Advanced Telemetry listening code, please temporarily hide this label using the `hidden` attribute. That way, nightly users won't ever see a checkbox that doesn't do anything.